.github/workflows/cmake-multi-platform.yml

```diff
@@ -71,8 +76,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+
+    - name: Create Git LFS file list
+      run: git lfs ls-files -l |cut -d' ' -f1 |sort >.git/lfs-hashes.txt
+
+    - name: Restore Git LFS object cache
+      uses: actions/cache@v4
       with:
-        lfs: 'true'
+        path: .git/lfs
+        key: ${{ runner.os }}-lfsdata-v1-${{ hashFiles('.git/lfs-hashes.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-lfsdata-v1-
+          ${{ runner.os }}-lfsdata
+
+    - name: Fetch any needed Git LFS objects and prune extraneous ones
+      run: git lfs fetch --prune
+
+    - name: Check out Git LFS content
```

Comment:
I looked at the output (your test runs) and it seems that we had a cache hit, but this line still outputs:

```
Checking out LFS objects: 100% (8/8), 107 MB | 0 B/s, done
```

Is that expected?

Reply:
ok, I need to check the if

here are the new run
https://github.com/FrankYFTang/inflection/actions/runs/15622526187

and
https://github.com/FrankYFTang/inflection/actions/runs/15622524628